Files.Launcher/Program.cs

```diff
@@ -328,6 +328,11 @@ await args.Request.SendResponseAsync(new ValueSet()
                 case "GetOneDriveAccounts":
                     using (var oneDriveAccountsKey = Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Microsoft\OneDrive\Accounts", false))
                     {
+                        if (oneDriveAccountsKey == null)
+                        {
+                            await args.Request.SendResponseAsync(new ValueSet());
```

Comment:
Missing `return` statement here? It will still go on with the rest of the code (and crash)

Reply:
Sorry my bad, fixed.

Reply:
@gave92, do you have any tips on debugging FullTrust when launching Files.Package? I find if Files is already deployed Files uses the deployed FullTrust version instead of the latest build with the VS code.

Reply:
Hitting compile > deploy solution will also update the fulltrust process.